include/multipass/platform.h

```diff
@@ -36,13 +36,22 @@ namespace multipass
 {
 namespace platform
 {
+std::map<QString, QString> extra_settings_defaults();
+
+QString interpret_winterm_integration(const QString& val);
+void sync_winterm_profiles();
```

Comment:
This interface should be agnostic to the fact that a platform may decide to do Windows Terminal integration.

```suggestion
QString interpret_setting(const QString& key, const QString& val);
void perform_final_adjustments();
```

Reply:
Agreed, that makes it more general.

Reply:
done